graphql/schemabuilder/helpers.go

```diff
@@ -269,46 +269,13 @@ func getConnection(res []interface{}, args ConnectionArgs) (interface{}, error)
 
 }
 
-func sliceAfter(edges []Edge, first int64, after *string) []Edge {
```

Comment:
i think the removal of `sliceAfter` and `sliceBefore` goes in the previous commit.